qiita_db/meta_util.py

```diff
@@ -555,20 +555,44 @@ def generate_plugin_releases():
         f(redis_key, v)
 
 
-def get_software_commands(active):
+def _build_software_commands_col_names_object(active):
```

Comment:
This is only called once and is not directly tested, why create a separate method for this?